pkg/arvo/sys/lull.hoon

```diff
@@ -2317,19 +2320,20 @@
       =>  |%
           ++  port  (aura 'udE')
           --
-      %+  pick
+      ;:  pick
         (bits & 256)
-      %+  pick
         :(both (just %if) (aura 'ifF') port)
-      :(both (just %is) (aura 'isH') port)
+        :(both (just %is) (aura 'isH') port)
+      ==
     ::
     ++  pactt
       ^-  $-(@uvJ [pact:pact @uvJ])
-      %+  pick
-        (both (just %peek) name)
-      %+  pick
-        :(both (just %page) name data (many [0 2] lane))
-      :(both (just %poke) name name data)
+      =/  hop  (rand 7)
+      ;:  pick
+        :(both hop (just %peek) name)
+        :(both hop (just %page) name data (many [0 2] lane))
+        :(both hop (just %poke) name name data)
+      ==
```

Comment:
It occurs to me that composing `+pick` in this way results in a significant bias: the first option has a 50% chance of being picked, while the next two only have a 25% chance each. To ensure that the odds are even, `+pick` needs to take a list and generate a random index:
```hoon
    ++  pick
      |*  gens=*
      %.  ^.(limo gens)
      |*  l=(list gen)
      |=  eny=@uvJ
      =^  a  eny  ((rand (lent l)) eny)
      ((snag a l) eny)
```
(@joemfb is there a cleaner way to write this? I tried the obvious approaches but they didn't work.)

Two choices is the most common case, so we probably want to keep an arm like that around, though I'm not sure what to name it -- `+pick2`? `+toss`? `+flip`?